app/lib/forms/choose_your_npq.rb

```diff
@@ -2,15 +2,33 @@ module Forms
   class ChooseYourNpq < Base
     include Helpers::Institution
 
-    attr_accessor :course_id
+    QUESTION_NAME = :course_identifier
 
-    validates :course_id, presence: true
+    attr_accessor QUESTION_NAME
+
+    validates QUESTION_NAME, presence: true
     validate :validate_course_exists
 
     def self.permitted_params
-      %i[
-        course_id
-      ]
+      [QUESTION_NAME]
+    end
+
+    def question
+      Forms::QuestionTypes::RadioButtonGroup.new(
+        name: :course_identifier,
+        options:,
+        style_options: { fieldset: { legend: { size: "m", tag: "h1" } } },
+      )
+    end
+
+    def options
+      divider_index = courses.length - 1 # Place the "Or" divider before the last course
```

Comment:
This still isn't great because it implicitly includes the assumption that the EHCO course is last, since the logic is that it's placed before EHCO (which is last) rather than before whichever NPQ is last.

Reply:
Is there a way of passing the 'or' option in separately? I think that'd make it easier to follow. Maybe as `options:, alternative_option:` or something?

Reply:
I understand this option makes it easy to change order through the DB rather than a code change although as i understand this doesnt really happen that often. I guess i still would have been a fan of having all the radio options listed here unless we think it looks messy? Its just trying to control things like the divider and anything else that might happen seems too complex for a simple list? I might be in the minority camp here though?

Reply:
I don't think it being a code change is necessarily a bad thing, so long as a new dev in a year's time can easily work out what's going on. However, there's no reason that can't be addressed as a follow up.